test/lib/ansible_test/_util/target/setup/requirements.py

```diff
@@ -349,18 +348,16 @@ def remove_tree(path):  # type: (str) -> None
     """Remove the specified directory tree."""
     try:
         shutil.rmtree(to_bytes(path))
-    except OSError as ex:
-        if ex.errno != errno.ENOENT:
-            raise
+    except FileNotFoundError:
+        pass
 
 
 def make_dirs(path):  # type: (str) -> None
     """Create a directory at path, including any necessary parent directories."""
     try:
         os.makedirs(to_bytes(path))
```

Comment:
Similarly to the updates at the top of this PR, this could be changed into `exists_ok=True`.